string a_id
simple_planning/position goal
---

